### PR TITLE
[Adjust] Report user’s anonymous ID as a callback parameter.

### LINKED ISSUE #2128: Add Adjust user identification
This is needed so that the marketing team can match up installations to campaigns.

We should use the anonymous user ID so that:
- They can have insight into what users end-up creating accounts (as we can already relate anonymous IDs to Gravity user IDs)
- They can have insight into what users end-up _not_ creating accounts.

### PR DESCRIPTION
**Pull request by alloy**
Closes #2128

### CODE REVIEW

File: Artsy/App/ARAppDelegate.m [hunk 1]
```diff
@@ -406,7 +406,9 @@ - (void)countNumberOfRuns
     NSInteger numberOfRuns = [[NSUserDefaults standardUserDefaults] integerForKey:ARAnalyticsAppUsageCountProperty] + 1;
     if (numberOfRuns == 1) {
         [ARAnalytics event:ARAnalyticsFreshInstall];
-        [Adjust trackEvent:[ADJEvent eventWithEventToken:ARAdjustFirstUserInstall]];
+        ADJEvent *event = [ADJEvent eventWithEventToken:ARAdjustFirstUserInstall];
+        [event addCallbackParameter:@"anonymous_id" value:ARUserManager.sharedManager.localTemporaryUserUUID];
```
Comment (alloy) [inline Artsy/App/ARAppDelegate.m]: So this event becomes something like:

```
{
    "anonymous_id" = "F01EFA7B-8479-474A-A31F-0605D24B76D0";
}
```

**Comment by orta** [comment]
Nice one, looks legit to me 👍

File: CHANGELOG.yml [hunk 2]
```diff
@@ -1,8 +1,16 @@
 upcoming:
-    version: 3.1.0
+    version: 3.1.1
```
Comment (orta) [inline CHANGELOG.yml]: wait, bah, this version didn't ship

Comment (orta) [inline CHANGELOG.yml]: @ashfurrow this will need reverting ^

Comment (alloy) [inline CHANGELOG.yml]: Ahhhh, forgot it influenced the automation, sorry about that.

Comment (ashfurrow) [inline CHANGELOG.yml]: No problem, I'm working around it 👍

**Pull request merged**